queries.sql

```diff
@@ -49,4 +49,35 @@ GROUP BY species;
 SELECT species, AVG(escape_attempts) as escape_attempts
 FROM animals
 WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
-GROUP BY species;
\ No newline at end of file
+GROUP BY species;
+
+SELECT name FROM animals
+JOIN owners ON animals.owner_id = owners.id
+WHERE owners.full_name = 'Melody Pond';
+
+SELECT * FROM animals
+INNER JOIN species ON  animals.species_id = species.id
+WHERE species.id = 1;
+
+SELECT owners.full_name, animals.name
+FROM owners
+LEFT JOIN animals ON owners.id = animals.owner_id;
```

Comment:
- From your screenshot, the result from this query is wrong, it shows `Sam smith` has no animal, but `Sam Smith` has `Agumon`, please fix this 🙏